example/glue/configure_includes/configure_linux_riscv.mk

```diff
@@ -21,17 +21,20 @@
 
 include $(CONFIG_INCL_DIR)/configure_common.mk
 
-RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
-
 ifneq (,$(findstring _riscv64, $(SPEC)))
-	RISCV_TARGET=riscv64-unknown-linux-gnu
+	RISCV_CROSSTOOLS_PREFIX ?= riscv64-unknown-linux-gnu
+	OMR_RISCV_CROSSTOOLS_PREFIX=$(RISCV_CROSSTOOLS_PREFIX)
 else
-	RISCV_TARGET=riscv32-unknown-linux-gnu
+	ifeq (install, $(RISCV_TOOLCHAIN_TYPE))
+		OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-linux-gnu
+	else
+		OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-unknown-linux-gnu
+	endif

```

Comment:
The only (non-continuation) lines that should begin with tabs are those that define the script of a rule and there are no rules in this file.

Why do we need both `RISCV_CROSSTOOLS_PREFIX` and `OMR_RISCV_CROSSTOOLS_PREFIX`?

Reply:
Replaced tabs with spaces as suggested above and replaced `OMR_RISCV_CROSSTOOLS_PREFIX `with `RISCV_CROSSTOOLS_PREFIX ` everywhere as it is unnecessary.